dracut.sh

```diff
@@ -2360,16 +2360,7 @@ fi
 
 if [[ $kernel_only != yes ]]; then
```

Comment:
Maybe it would be enough just to move this if under "if dracut_module_included "squash"; then" one, but in the end it does not matter much and since this the upstream solution I am fine with it